src/state/notifications/reducer.ts

```diff
@@ -81,3 +93,8 @@ export const notificationReducer = (state = initialState, action: NotificationAc
       return state;
   }
 };
+
+export const notificationReducer = createPersistReducer(reducer, {
```

Comment:
What's the gain in specifying separated reduces for notifications?

Reply:
that we don't save trash information, check the PR description for more info